contents/docs/api/query/query_create.mdx

```diff
@@ -0,0 +1,82 @@
+This is the main endpoint for querying data from PostHog. It is used for many kinds of queries including `TrendsQuery`, `FunnelsQuery`, `ActorsQuery`, `EventsQuery`, and more.
```

Comment:
My initial thought is what is `TrendsQuery`, `FunnelsQuery`, etc. and where can I learn more about these/see the full list. Can we hyperlink to somewhere?